### ISSUE TITLE
Error forwarding the new session Error forwarding the request Read timed out

### ISSUE THREAD

**Issue by guilhotarras** [SeleniumHQ/selenium#1922]
I'm using selenium grid 2.53 to run my tests in parallel and sometimes I have this error:

> 06:45:14.729 [chrome #01-32] PID: 63969
06:45:14.729 [chrome #01-32] Specs: /var/lib/go-agent/pipelines/FrontEnd-TVG4-Dev/workspace/tvg-tvg4-angular/tests/web-tests/mock-tests/tests/account/desktop/my-profile-page-update-profile-3-spec.js
06:45:14.729 [chrome #01-32] 
06:45:14.730 [chrome #01-32] Using the selenium server at http://10.216.1.49:4444/wd/hub
06:45:14.730 [chrome #01-32] 
06:45:14.730 [chrome #01-32] /var/lib/go-agent/pipelines/FrontEnd-TVG4-Dev/workspace/tvg-tvg4-angular/tests/web-tests/node_modules/selenium-webdriver/error.js:26
06:45:14.730 [chrome #01-32]   constructor(opt_error) {
06:45:14.730 [chrome #01-32]                          ^
06:45:14.730 [chrome #01-32] WebDriverError: Error forwarding the new session Error forwarding the request Read timed out
06:45:14.730 [chrome #01-32]     at WebDriverError (/var/lib/go-agent/pipelines/FrontEnd-TVG4-Dev/workspace/tvg-tvg4-angular/tests/web-tests/node_modules/selenium-webdriver/error.js:26:26)
06:45:14.730 [chrome #01-32]     at Object.checkLegacyResponse (/var/lib/go-agent/pipelines/FrontEnd-TVG4-Dev/workspace/tvg-tvg4-angular/tests/web-tests/node_modules/selenium-webdriver/error.js:580:13)
06:45:14.730 [chrome #01-32]     at /var/lib/go-agent/pipelines/FrontEnd-TVG4-Dev/workspace/tvg-tvg4-angular/tests/web-tests/node_modules/selenium-webdriver/lib/webdriver.js:64:13
06:45:14.730 [chrome #01-32]     at Promise.invokeCallback_ (/var/lib/go-agent/pipelines/FrontEnd-TVG4-Dev/workspace/tvg-tvg4-angular/tests/web-tests/node_modules/selenium-webdriver/lib/promise.js:1329:14)
06:45:14.730 [chrome #01-32]     at TaskQueue.execute_ (/var/lib/go-agent/pipelines/FrontEnd-TVG4-Dev/workspace/tvg-tvg4-angular/tests/web-tests/node_modules/selenium-webdriver/lib/promise.js:2790:14)
06:45:14.730 [chrome #01-32]     at TaskQueue.executeNext_ (/var/lib/go-agent/pipelines/FrontEnd-TVG4-Dev/workspace/tvg-tvg4-angular/tests/web-tests/node_modules/selenium-webdriver/lib/promise.js:2773:21)
06:45:14.730 [chrome #01-32]     at /var/lib/go-agent/pipelines/FrontEnd-TVG4-Dev/workspace/tvg-tvg4-angular/tests/web-tests/node_modules/selenium-webdriver/lib/promise.js:2652:27
06:45:14.730 [chrome #01-32]     at /var/lib/go-agent/pipelines/FrontEnd-TVG4-Dev/workspace/tvg-tvg4-angular/tests/web-tests/node_modules/selenium-webdriver/lib/promise.js:639:7
06:45:14.730 [chrome #01-32]     at process._tickCallback (node.js:405:9)
06:45:14.730 [chrome #01-32] From: Task: WebDriver.createSession()
06:45:14.730 [chrome #01-32]     at acquireSession (/var/lib/go-agent/pipelines/FrontEnd-TVG4-Dev/workspace/tvg-tvg4-angular/tests/web-tests/node_modules/selenium-webdriver/lib/webdriver.js:62:22)
06:45:14.731 [chrome #01-32]     at Function.createSession (/var/lib/go-agent/pipelines/FrontEnd-TVG4-Dev/workspace/tvg-tvg4-angular/tests/web-tests/node_modules/selenium-webdriver/lib/webdriver.js:295:12)
06:45:14.731 [chrome #01-32]     at Builder.build (/var/lib/go-agent/pipelines/FrontEnd-TVG4-Dev/workspace/tvg-tvg4-angular/tests/web-tests/node_modules/selenium-webdriver/builder.js:458:24)
06:45:14.731 [chrome #01-32]     at [object Object].DriverProvider.getNewDriver (/var/lib/go-agent/pipelines/FrontEnd-TVG4-Dev/workspace/tvg-tvg4-angular/tests/web-tests/node_modules/protractor/built/driverProviders/driverProvider.js:42:27)
06:45:14.731 [chrome #01-32]     at [object Object].Runner.createBrowser (/var/lib/go-agent/pipelines/FrontEnd-TVG4-Dev/workspace/tvg-tvg4-angular/tests/web-tests/node_modules/protractor/built/runner.js:203:37)
06:45:14.731 [chrome #01-32]     at /var/lib/go-agent/pipelines/FrontEnd-TVG4-Dev/workspace/tvg-tvg4-angular/tests/web-tests/node_modules/protractor/built/runner.js:293:21
06:45:14.731 [chrome #01-32]     at _fulfilled (/var/lib/go-agent/pipelines/FrontEnd-TVG4-Dev/workspace/tvg-tvg4-angular/tests/web-tests/node_modules/q/q.js:834:54)
06:45:14.731 [chrome #01-32]     at self.promiseDispatch.done (/var/lib/go-agent/pipelines/FrontEnd-TVG4-Dev/workspace/tvg-tvg4-angular/tests/web-tests/node_modules/q/q.js:863:30)
06:45:14.731 [chrome #01-32]     at Promise.promise.promiseDispatch (/var/lib/go-agent/pipelines/FrontEnd-TVG4-Dev/workspace/tvg-tvg4-angular/tests/web-tests/node_modules/q/q.js:796:13)
06:45:14.731 [chrome #01-32]     at /var/lib/go-agent/pipelines/FrontEnd-TVG4-Dev/workspace/tvg-tvg4-angular/tests/web-tests/node_modules/q/q.js:556:49
06:45:14.731 
06:45:14.731 [launcher] Runner process exited unexpectedly with error code: 1
06:45:14.732 [launcher] 3 instance(s) of WebDriver still running

My file to start the HUB:

ExecStart=/bin/java -Xmx1g \
            -Xms64m \
            -Dvideo.storage=com.aimmac23.hub.videostorage.LocalFileVideoStore \
            -Dvideo.path=/opt/selenium/screencast \
            -cp \
            /opt/selenium/selenium-server-standalone.jar \
            org.openqa.grid.selenium.GridLauncher \
            -role hub \
            -DPOOL_MAX=512 \
            -port 4444 -timeout 60 -browserTimeout 60

My file to start the NODE (I have 5 nodes): 

ExecStart=/usr/bin/xvfb-run -a -s \
           '-screen 1 1280x1024x24 -wr -fbdir /tmp/screen_1' \
            /bin/java -Xmx512m \
            -cp \
            /opt/selenium/selenium-server-standalone.jar\
            org.openqa.grid.selenium.GridLauncher \
            -role node \
            -Dwebdriver.chrome.driver=/opt/selenium/drivers/chromedriver \
            -nodeConfig /etc/selenium/node.json \
            -hubHost 10.216.1.49 \
            -hubPort 4444 \
            -port 5556

My node.config file:

{
    "ATTENTION" : "This file was generated by Chef. Do NOT modify this file by hand.",
    "capabilities": [
        {
            "browserName": "chrome",
            "version": "49.0.2623.110-1",
            "maxInstances": 3,
            "seleniumProtocol": "WebDriver",
            "platform": "LINUX",
            "applicationName": "general"
        },
        {
          "browserName": "firefox",
          "version": "38.6.0-1.el7.centos",
          "maxInstances": 3,
          "seleniumProtocol": "WebDriver",
          "platform": "LINUX",
          "applicationName": "general"
        }
    ],
    "configuration": {
        "proxy": "org.openqa.grid.selenium.proxy.DefaultRemoteProxy",
        "maxSession": 3,
        "register": true,
        "registerCycle": 5000,
        "versions": {
          "chromeDriver": 2.21,
          "seleniumServer": 2.53
        }
    }
}

Can you help? 

Thanks,

**Comment by pmpranav** [comment]
Facing the same issue

**Comment by theandrewlane** [comment]
Me too!

**Comment by theandrewlane** [comment]
I'm running into the same thing too.

**Comment by barancev** [comment]
Duplicate of #1261
